### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by chaihuoa** [AppScaffold/ASDebugger#1]
![screen shot 2019-02-15 at 3 51 12 pm](https://user-images.githubusercontent.com/9607316/52842614-9c339080-313a-11e9-8bbf-cd82649aeafb.png)

Get an error 'Can not find `AFCompatibilityMacros`' when I run this project first after download.

**Comment by chaihuoa** [comment]
I have fixed this after I removed the `Pods` and reinstall it. But maybe we don't need to commit these files in `Pods/`. @squarezw Can I pull a request to delete the `Pods` and ignore it?

**Comment by squarezw** [comment]
true, let's remove `Pods` directory. seems it's unnecessary at all.

**Comment by chaihuoa** [comment]
Thanks, I have pulled a merge request https://github.com/AppScaffold/ASDebugger/pull/2
please help me review it.

**Comment by chaihuoa** [comment]
Sorry for some legacy pod files haven't been deleted. Please help me review this PR https://github.com/AppScaffold/ASDebugger/pull/3, thanks.
